rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml

```diff
@@ -27,7 +27,8 @@ fields:
     - CommandLine
     - ParentCommandLine
 falsepositives:
-    - Unknown
+    - During Anaconda update the 'conda.exe' process will eventually launch the command described in the detection section
+    - Other currently unknown cases
```

Comment:
we don't need to explain that there are unknown unknowns, I think

Reply:
Agreed